deepeval/metrics/hallucination_metric.py

```diff
@@ -0,0 +1,43 @@
+import os
+from deepeval.singleton import Singleton
+from deepeval.test_case import LLMTestCase
+from deepeval.utils import chunk_text, softmax
+from deepeval.metrics.base_metric import BaseMetric
+from deepeval.evaluator import assert_test
+from deepeval.progress_context import progress_context
+from sentence_transformers import CrossEncoder
```

Comment:
I guess, we do not need this import any more. So, I believe this can be removed.